DigitalLearningSolutions.Web/Controllers/HomeController.cs

```diff
@@ -63,24 +64,30 @@ public IActionResult LearningContent()
         {
             var publicBrands = brandsService.GetPublicBrands()
                 .Select(b => new LearningContentSummary(b));
-
             var model = new LearningContentLandingPageViewModel
             {
                 MiniHubNavigationModel = new MiniHubNavigationModel(LandingPageMiniHubName, sections, 2),
                 UserIsLoggedIn = User.Identity.IsAuthenticated,
+                UserIsLoggedInCentre = GetCentreId() != null,

```

Comment:
See below comment, this can call the existing custom claims helper method `UserIsLoggedInCentre = User.GetCentreId() != null,`